### PR TITLE
Mark comment as private

### LINKED ISSUE #11: Mark private // 8
In order to privately post a comment as I user i want toe be able to mark a comment as private
- [x] Create a comment model with a private flag property
- [x] Create an add comment action in the idea controller
- [x] Implement comment abilities in the idea view
- [ ] Implement ACL

### PR DESCRIPTION
**Pull request by dennisdegreef**
Closes #11

### CODE REVIEW

File: src/AppBundle/Controller/IdeaController.php [hunk 1]
```diff
@@ -35,9 +40,16 @@ public function createIdeaAction()
 
     }
 
-    public function createCommentAction()
+    public function createCommentAction($id)
     {
+        $private = false;
+
+        if(isset($_POST['private']) && $_POST['private'] === 'on') {
```
Comment (piwi91) [inline src/AppBundle/Controller/IdeaController.php]: Use the `Request` object please :)

File: src/AppBundle/Controller/IdeaController.php [hunk 2]
```diff
@@ -35,9 +41,16 @@ public function createIdeaAction()
...
-    public function createCommentAction()
+    public function createCommentAction($id, Request $request)
     {
+        $private = false;
+
+        if($request->get('private') === 'on') {
+            $private = true;
+        }
 
+        $this->get('idea.service')->comment($id, $_POST['comment'], $private);
```
Comment (ussuritiger) [inline src/AppBundle/Controller/IdeaController.php]: Can we get a comment from Request object?

**Comment by ussuritiger** [comment]
:+1:

File: src/AppBundle/Controller/IdeaController.php (hunk 2 above)
Comment (dennisdegreef) [inline src/AppBundle/Controller/IdeaController.php]: Fixed

**Comment by dennisdegreef** [comment]
LGTM? :)

**Comment by piwi91** [comment]
LGTM :+1:

**Comment by ussuritiger** [comment]
@dennisdegreef yeap

**Pull request merged**